jobs/ci-kubernetes-e2e-gce-scalability-canary.env

```diff
@@ -0,0 +1 @@
+# place-holder
```

Comment:
So we are currently assuming each job has a matching job.env file, and there are places we iterating .env files rather than the actual jobs. Using same env file is fine. We'd need to do a pass to clean up those env files, hopefully after we remove most of the global envs.